### PR TITLE
Translate the readme

### PR DESCRIPTION
**Pull request by cendekia**

### CODE REVIEW

**Comment by tesseralis** [comment]
@coroo could you review this? and if you're not in the gatsbyjs org yet, could you accept the invitation?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,11 @@
...
-Useful Links:
+Tautan Bermanfaat:
 
-* [Style Guide](/style-guide.md)
-* [Gatsby Translation Guide](https://www.gatsbyjs.org/contributing/gatsby-docs-translation-guide/)
-
-(Feel free to translate this document and add any content you feel would be useful to contributors).
-  
\ No newline at end of file
+* [Panduan Style](/style-guide.md)
```
Comment (coroo) [inline README.md]: Its better to use: "Panduan Penulisan"

**Review by coroo — CHANGES_REQUESTED** [review]
Please completed my comment above, thank you

**Comment by cendekia** [comment]
@coroo 
https://github.com/gatsbyjs/gatsby-id/pull/2/commits/3a47b5529eed8796c4396537fa31a59342e92796

Done :)))

**Review by coroo — APPROVED** [review]

**Pull request merged**